about/index.html

```diff
@@ -2,4 +2,10 @@
 layout: simple
 primary_title: About
 title: About
----
\ No newline at end of file
+---
+
+
+
+<h1>History of Valkey</h1>
+
+Valkey is an open source fork of Redis, created in March 2023 after Redis announced a change to a more <a href="https://redis.io/blog/redis-adopts-dual-source-available-licensing/">restrictive license</a>. The project was initiated by <a href="https://github.com/madolson">Madelyn Olson</a>, a former Redis core maintainer from AWS, along with other Redis contributors who disagreed with the license change. Valkey quickly gained support from major tech companies such as AWS, Google, Oracle, Ericsson, and Snap, with the Linux Foundation providing a new home for the fork to maintain the original open source license. Despite the emergence of other Redis alternatives and forks, Valkey has gained significant traction due to its strong backing from cloud companies that rely heavily on Redis for their services, making it the most promising option for the near future.
```

Comment:
- There are a lot of opinions about the word 'fork'. Maybe: `Valkey is a open source continuation of Redis created`
- `more restrictive license` change to `non-open source license`.